src/forking-store.js

```diff
@@ -229,6 +234,10 @@ export default class ForkingStore {
     return [...forGraphs];
   }
 
+  isDirty() {
```

Comment:
Nitpick: I would make it a getter, which seems nicer to use?
```suggestion
  get isDirty() {
```

Extra nitpicky: I'm not sure about the name. What do you think about `hasChanges`? I know it's an EmberData model thing, but I never liked the naming there either :sweat_smile: